src/main/resources/db/migration/V43_47__Create_community_zone_table.sql

```diff
@@ -0,0 +1,7 @@
+create table community_zone
```

Comment:
Rename it to community_authorized_zone and same as the classes, not only the attributes. 

Don't forget that a bad name for the table and class can handle to error when someone extern from the project as a reviewer (as me) read it. Be the much explicit as possible.